lib/mayaUsd/undo/UsdUndoableItem.h

```diff
@@ -14,16 +14,14 @@
 // limitations under the License.
 //
 
-#ifndef MAYAUSD_UNDO_UNDOABLE_BLOCK_H
-#define MAYAUSD_UNDO_UNDOABLE_BLOCK_H
+#ifndef MAYAUSD_UNDO_UNDOABLE_ITEM_H
+#define MAYAUSD_UNDO_UNDOABLE_ITEM_H
 
 #include <mayaUsd/base/api.h>
 
 #include <functional>
 #include <vector>
 
-PXR_NAMESPACE_USING_DIRECTIVE
```

Comment:
No Pixar namespace declarations or definitions here, removed.

Reply:
love it